airflow/models/edgemodifier.py

```diff
@@ -0,0 +1,126 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Sequence, Union
+
+from airflow.models.taskmixin import TaskMixin
+
+
+class EdgeModifier(TaskMixin):
```

Comment:
It isn't a model so we should probably move it to some other place instead of inside `airflow.models` -- probably inside `airflow/models/baseoperator.py` where we also have `set_upstream` and `set_downstream`?

Reply:
Interesting, I was going off of `XComArg` having its own file rather than being in `xcom.py`, but happy to move it. Maybe baseoperator, maybe it should live alongside `taskgroups` in utils since it serves a similar UI-only purpose?